<launch>
    <arg name="velTopicArg" default="/cmd_vel" />
    <arg name="chassisSerialNameArg" default="/dev/ttyUSB0" />
    <arg name="chassisSerialBaudArg" default="115200" />
    <param name="velTopic" value="$(arg velTopicArg)" />
    <param name="chassisSerialName" value="$(arg chassisSerialNameArg)" />
    <param name="chassisSerialBaud" value="$(arg chassisSerialBaudArg)" />
    <node pkg="chassis" type="chassis_node" name="chassis_node" output="screen" />
</launch>
